docs/server/features.md

```diff
@@ -0,0 +1,105 @@
+# Features
+
+This section provides a detailed overview of the features provided by the Ruff Language Server.
+
+## Diagnostic Highlighting
+
+Provide diagnostics for your Python code in real-time.
+
+<img
+src="https://astral.sh/static/GIF/v0.4.5/violation_hx.gif"
+alt="Editing a file in Helix"
+/>
+
+## Dynamic Configuration
+
+The server can dynamically refresh the diagnostics when a configuration file is changed in the
+workspace. This could be either a `pyproject.toml` file or a `ruff.toml` / `.ruff.toml` file.
```

Comment:
"in the workspace, whether it's a `pyproject.toml`, `ruff.toml`, or `.ruff.toml` file."